include/configs/mx6_cubox-i.h

```diff
@@ -65,7 +65,7 @@
 #define CONFIG_CMD_SETEXPR
 #define CONFIG_CMD_MEMTEST
 
-#define CONFIG_BOOTDELAY		1
+#define CONFIG_BOOTDELAY		3
```

Comment:
Increased the boot delay to 3s to give more time to copy-paste the interruption command.  Not sure if it is really needed, more of a convenience.